presto-main/src/main/java/com/facebook/presto/server/StatementResource.java

```diff
@@ -573,11 +573,11 @@ private synchronized URI createNextResultsUri(UriInfo uriInfo)
 
         private static List<Column> createColumnsList(QueryInfo queryInfo)
         {
-            requireNonNull(queryInfo, "queryInfo is null");
-            checkArgument(queryInfo.getOutputStage().isPresent(), "outputStage is not present");
+            StageInfo outputStage = queryInfo.getOutputStage()
+                    .orElseThrow(() -> new IllegalArgumentException("outputStage not present"));
```

Comment:
IDEA warns that `get()` is called without a check. It doesn't like things like this:
```java
checkArgument(object.getOptionalField().isPresent());
object.getOptionalField.get();
```
Arguably, it should do a better job for getters that return a final field.